get-started/codelab/index.md

```diff
@@ -207,92 +206,83 @@ You can find the
 package, as well as many other open source packages, on
 [pub.dartlang.org](https://pub.dartlang.org/flutter/).
 
-<ol markdown="1">
-
-<li markdown="1"> The pubspec file manages the assets for a Flutter app.
+ 1. The pubspec file manages the assets for a Flutter app.
     In **pubspec.yaml**, add **english_words** (3.1.0 or higher)
     to the dependencies list.
     The new line is highlighted below:
 
-<!-- skip -->
-{% prettify dart %}
-dependencies:
-  flutter:
-    sdk: flutter
+    {% prettify yaml %}
+      dependencies:
+        flutter:
+          sdk: flutter
 
-  cupertino_icons: ^0.1.0
-  [[highlight]]english_words: ^3.1.0[[/highlight]]
-{% endprettify %}
-</li>
+        cupertino_icons: ^0.1.0
+        [[highlight]]english_words: ^3.1.0[[/highlight]]
+    {% endprettify %}
 
-<li markdown="1"> While viewing the pubspec in Android Studio's editor view,
+ 2. While viewing the pubspec in Android Studio's editor view,
     click **Packages get** upper right. This pulls the package into
     your project. You should see the following in the console:
 
-<!-- skip -->
-{% prettify dart %}
-flutter packages get
-Running "flutter packages get" in startup_namer...
-Process finished with exit code 0
-{% endprettify %}
-</li>
+    ```terminal
+    > flutter packages get
+    Running "flutter packages get" in startup_namer...
+    Process finished with exit code 0
+    ```
```

Comment:
Is the use of this new terminal styling ok (set green on black)?

Reply:
If it's consistent with dartlang, that's fine. (I haven't seen it yet, though.)

Reply:
Yes it is consistent with dartlang, e.g., see https://www.dartlang.org/tools/sdk#install

Reply:
Then leave it.  :)